backend/src/handlers_prelude/github_pull_request.rs

```diff
@@ -0,0 +1,54 @@
+//! Github Pull Request are sent here.
+use axum::{extract::State, http::StatusCode, Json, Router};
+use axum::routing::post;
+use tracing::{error, info};
+use serde_json::json;
+use crate::gh::create_pull_request;
+use crate::AppState;
+use color_eyre::Result;
+
+#[derive(serde::Deserialize, Debug)]
+pub struct CreatePRRequest {
+    pub head_branch: String,    // The branch containing changes
```

Comment:
If these comments were docstrings on the function field then we'd get fancy ide shit